### ISSUE TITLE
FileSystemAdapter outputs files as directories

### ISSUE THREAD

**Issue by FROGGS** [sciactive/nephele#8]
Hi, I've subclassed the VirtualAdapter to serve stuff from an sql database as files and folders which is awesome, but the FileSystemAdapter gives me some headache.
In Folder bar are some files, but these appear as folders. I can even open these folders but of course the appear empty.
I've tried to debug the adapter, but with no luck so far. The resources start being not a directory, but somewhere down the road their url have a traling slash and so are treated as a directory. Though, this might happen because my file explorer tries to PROFIND these "directories".
I hope you can reproduce the issue, it is already late where I live. If you need more information I'll provide these tomorrow happily.

```typescript
createServer({
      adapter: {
        '/foo/': new FileSystemAdapter({ root: './mount/foo/' }),
        '/bar/': new FileSystemAdapter({ root: './mount/bar/' }),
        '/': new VirtualAdapter({ ... }),,
      },
      authenticator: new InsecureAuthenticator(),
      plugins: [],
    })
```

**Comment by hperrin** [comment]
Can you tell me which OS you're using?

**Comment by FROGGS** [comment]
The client runs Windows 10 (Windows Explorer), the server is Ubuntu 22.04.

**Comment by hperrin** [comment]
Ok. Which file system are you using? I'll try to recreate the issue locally.

**Comment by FROGGS** [comment]
I just found out, that only the first directory level is affected.

1. I create a folder inside `bar`
2. I create a file inside the new folder using the webdav client, press F5, everything in okay
3. I create a file inside bar directly, press F5, and the file now looks like a folder.

Filesystem is ext4 on server side.

**Comment by hperrin** [comment]
Ok. Let me try to reproduce this.

**Comment by hperrin** [comment]
Yep, I'm able to reproduce this. I'll work on a fix.

**Comment by FROGGS** [comment]
Awesome, thank you very much.

**Comment by hperrin** [comment]
Fixed and released as 1.0.0-alpha.61.
